pom.xml

```
@@ -74,7 +74,7 @@
     <jackson.datatype.groupId>com.fasterxml.jackson.datatype</jackson.datatype.groupId>
     <jackson.package>com.fasterxml.jackson</jackson.package>
     <jackson.version>2.13.2</jackson.version>
```

The jackson version should be updated as well so the versions match.
```suggestion
    <jackson.version>2.13.4</jackson.version>
```